src/board/internal.tsx

```diff
@@ -221,7 +229,12 @@ export function InternalBoard<D>({
         onActivateItem={focusItem}
       />
 
-      <div ref={containerRef} className={clsx(styles.root, { [styles.empty]: rows === 0 })}>
+      <div
+        ref={containerRef}
+        className={clsx(styles.root, currentOperation && styles[`current-operation-${currentOperation}`], {
```

Comment:
User may drag the widget outside the board area. I think, we should apply the new class to the entire document to handle this

Reply:
I was debating this too... I'll change it to apply on the body instead then.